keras_cv/layers/preprocessing/random_gaussian_blur.py

```diff
@@ -61,7 +61,9 @@ def __init__(self, kernel_size, factor, **kwargs):
                 )
 
     def get_random_transformation(self, **kwargs):
-        factor = self.factor()
+        # `factor` must not become too small otherwise numerical issues occur.
+        # keras.backend.epsilon() behaves like 0 without causing `nan`s
+        factor = max(self.factor(), keras.backend.epsilon())
```

Comment:
Looks like this is breaking tests -- probably because `self.factor()` returns a tf Tensor and `epsilon` is a python float.

Perhaps better would be to say `factor = self.factor() + keras.backend.epsilon()`

You can run the test locally to verify that this isn't breaking

Reply:
It would be very convenient if the tests ran on each update of the pull request automatically. I think the issue could be fixed now. Can you rerun the tests?

Reply:
Tested it locally, it seems to work now. I think the problem was that tensorflow has problems compiling the  `max`.

Reply:
They do run automatically for folks who have contributed before. However, you should always run tests locally instead of depending on CI to run tests

Reply:
Great! Hopefully CI will be happy here as well.

When using `max` in TF you need to use the tf-built in version, not the Python version, if you want to use TF tensors and have it compile into the TF graph. So in this case we could have used the TF version, but just adding the constant value also works just fine 😄

Reply:
In the meantime I tested it locally and it worked. There is no documentation on how to run the tests but it is not so hard to figure out. Could be improved though.

Reply:
CI tests look good to me except some unrelated formatting issue in some other code.

Reply:
Agreed! The formatting issue is unrelated and I have a fix open in #1993.

We have some (limited) guidance on running tests [here](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md#run-tests)